staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -166,3 +166,47 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PriorityAndFairnessConfiguration holds API Priority and Fairness parameters.
+type PriorityAndFairnessConfiguration struct {
+	metav1.TypeMeta
+
+	// WorkEstimatorConfig holds work estimator parameters.
+	WorkEstimatorConfiguration WorkEstimatorConfiguration
+}
+
+// WorkEstimatorConfiguration holds work estimator parameters.
+type WorkEstimatorConfiguration struct {
+	// ListWorkEstimatorConfiguration holds work estimator parameters related to list requests.
+	ListWorkEstimatorConfiguration ListWorkEstimatorConfiguration
```

Comment:
What happened to https://github.com/kubernetes/kubernetes/pull/111222#discussion_r933056627 ?  The field names are long again.

Reply:
Oh, for some reason I thought we were talking about JSON tags. Renamed the fields.

Reply:
Both the JSON tags and the Go language field names are relevant.  Both are in the context of a "work estimator configuration" and so do not need to repeat those words.